pkg/kubectl/cmd/patch_test.go

```diff
@@ -28,7 +28,7 @@ import (
 	"k8s.io/kubernetes/pkg/kubectl/scheme"
 )
 
-func TestPatchObject(t *testing.T) {
+func DONTTestPatchObject(t *testing.T) {
```

Comment:
???

Reply:
hm, forgot to take this out, was using this to isolate test failures :)